worlds/sc2/PoolFilter.py

```diff
@@ -235,6 +235,7 @@ def generate_reduced_inventory(self, inventory_size: int, mission_requirements:
         """Attempts to generate a reduced inventory that can fulfill the mission requirements."""
         inventory: List[Item] = list(self.item_pool)
         locked_items: List[Item] = list(self.locked_items)
+        enable_morphling = get_option_value(self.world, "enable_morphling") == EnableMorphling.option_true
```

Comment:
I'd recommend using `self.world.options.enable_morphling` here, for mypy / typing reasons. The expression returns a `EnableMorphling` type (use `self.world.options.enable_morphling.value` if you specifically need an int). `get_option_value()` is a legacy function from before the world.options interface was added, the only advantage it has is dealing with world being null (which it shouldn't be here).